recipes/py-open-fonts/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "py-open-fonts" %}
+{% set version = "0.4.9" %}
+{% set dep_ver = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3bda536869bc4e67f76b1db7553ab0f142f64d41e5838d57b0f96dd98e0b4ea
+  patches:
+    - conda.patch
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - open-fonts-css = openfonts.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - open-fonts ={{ dep_ver }}
+    - click
+
+test:
+  imports:
+    - openfonts
+  commands:
+    - open-fonts-css Symbola
+    - open-fonts-css --help
+
+about:
+  home: https://github.com/kiwi0fruit/py-open-fonts
+  license: "Aggregate (Apache 2.0, SIL OFL 1.1, Public Domain, DejaVu Fonts)"
+  license_family: OTHER
```

Comment:
Can you please change the license_family to PUBLIC-DOMAIN? Otherwise, the recipe looks good to me.

Reply:
@npavlovikj No I can't. The license of the package is  not a Public domain hence it doesn't belong to PUBLIC-DOMAIN family. That would be a lie.

Reply:
@npavlovikj Is there an umbrella license family for FOSS licenses (free and open source)? I searched for one but didn't found. If there is one like that then I would gladly switch.

Reply:
Ah, thanks for clarifying that - I missed the "Aggregate" part.

Since there are multiple licenses, can you please try adding them in a list:
`license: Apache-2.0 AND SIL OFL 1.1 AND Public Domain AND DejaVu Fonts `
This is an example PR, https://github.com/conda-forge/staged-recipes/pull/10329/files.

Leaving the family type to OTHER is fine in this case, looks like FOSS fails under that, https://github.com/conda/conda-build/blob/7dac1cbad5195f719bde1eaeb0d5795186dc0eb0/tests/test_license_family.py#L130.

Reply:
@npavlovikj I've fixed this recipe and it's now MIT. Please review and merge.

Actually this (py-open-fonts) conda package only contains MIT licensed code. Complex fonts licenses are already packaged in open-fonts conda package. I consider changing it's recipe but I'm not sure. Please see this issue: https://github.com/conda-forge/open-fonts-feedstock/issues/1

Reply:
@npavlovikj Nevermind open-fonts conda package license issue: I've already fixed it thanks to your suggestion.

Please review this py-open-fonts recipe that is now MIT only (as it only contains MIT source code).